apps/dynatrace/base/kustomize.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: dynatrace
+  namespace: flux-system
+spec:
+  interval: 10m
+  path: ./apps/dynatrace/${ENVIRONMENT}/base
+  dependsOn:
+    - name: dynatrace-crd
+  postBuild:
+    substitute:
+      NAMESPACE: "dynatrace"
+      TEAM_NOTIFICATION_CHANNEL: "platops-build-notices"
```

Comment:
I think on this, could we create an environmenet variable for `platops-build-notices`? Moving forward with Flux v2 we have other apps that will reference this such as plum/camunda etc

Reply:
@hannah38 @nickmoyni  what you think?

Reply:
trivial to find replace in the repo 👅